rules/scala.bzl

```diff
@@ -90,6 +90,7 @@ _compile_attributes = {
             ".java",
             ".srcjar",
         ],
+        flags = ["DIRECT_COMPILE_TIME_INPUT"],
```

Comment:
After reading documentation, I'm not very clear on what this does. What does it do?